### PR TITLE
Add getCroppingRect method & export getScaleFromCropRect for image editor

### PR DESCRIPTION
**Pull request by chenesan**
# Purpose

- 讓 ImageEditor 暴露 `getCroppingRect` 方法，讓外面隨時取得裁切資訊
- 將 `getInitScale` 換名為 `getScaleFromInitCropRect`，並 export 出去

# Changes

- a list of what have been done
- maybe some code change

# Risk

Usually none, if you have any please write it here.

# TODOs

- [ ] Describe what should be done outside of this PR
- [ ] Maybe in other PRs or some manual actions.

### CODE REVIEW

File: packages/imageeditor/src/ImageEditor.js [hunk 1]
```diff
@@ -0,0 +1,265 @@
+import React, { PureComponent } from 'react';
```
Comment (chenesan) [inline packages/imageeditor/src/ImageEditor.js:1]: 將原有的 index.js 換名為 ImageEditor.js

File: packages/imageeditor/src/ImageEditor.js [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+      return this.editorRef.current.getImageScaledToCanvas();
+    }
+
+    getCroppingRect = () => {
+      if (!this.editorRef.current) {
+        return null;
+      }
+
+      return this.editorRef.current.getCroppingRect();
+    }
```
Comment (chenesan) [inline packages/imageeditor/src/ImageEditor.js:129]: 這支 PR 唯一新增的方法，目的只是多提供 avatar editor 原本就有的 `getCroppingRect`

File: packages/imageeditor/src/index.js [hunk 3]
```diff
@@ -1,257 +1,6 @@
...
-          <div className={BEM.canvas.toString()}>
-            {shouldShowPlaceholder ? placeholder : canvas}
-          </div>
-
-          {this.renderControl()}
-        </div>
-      );
-    }
-}
+export { default as getScaleFromCropRect } from './utils/getScaleFromCropRect';
```
Comment (chenesan) [inline packages/imageeditor/src/index.js:4]: index.js 改為 export

**Review by zhusee2 — APPROVED** [review]
LGTM

**Review by tz5514 — APPROVED** [review]

**Pull request merged**